src/main/python/deep_qa/training/text_trainer.py

```diff
@@ -74,12 +74,11 @@ def __init__(self, params: Dict[str, Any]):
         # one of the keys in `encoders`) to determine the type of the encoder, then pass the
         # remaining args to the encoder constructor.
         # Hint: Use lstm or cnn for sentences, treelstm for logical forms, and bow for either.
-        self.encoder_params = params.pop('encoder', {'default': {}})
+        self.encoder_params = params.pop('encoder', {'default': {}, 'word': {}})
```

Comment:
I don't think this is necessary.  Instead, I would implement some logic in `_get_encoder` around what to do if the requested key (like `'word'`) is not present.